get-random-throw.js

```diff
@@ -0,0 +1,46 @@
+export function getRandomThrow() {
+  //generate a random number between 1 and 3
+  const rpsRandomNumber = Math.ceil((Math.random() * 3));
+  
+  if (rpsRandomNumber === 1) {
+    return "rock";
+  } else if (rpsRandomNumber === 2) {
+    return "paper";
+  } else if (rpsRandomNumber === 3) {
+    return "scissors";
+  }
+}
+
+export function checkResults(player, computer) {
+  
+  if (player === computer) {
+    // I don't care what the different choices are if they're the same
+    // If the choices are the same, return a tie condition.
+    // Otherwise, check for either a win or loss condition only
+    return "draw";
+  } else if (player === "rock") {
+      if (computer === "scissors") {
```

Comment:
nested if/else logic can get hard to read and reason about. consider flattening this:

```js
if(player === computer) return 'draw';
if(player === 'rock' && computer === 'scissors')  return 'win';
if(player === 'rock' && computer === 'paper')  return 'loss';
if(player === 'paper' && computer === 'rock')  return 'win';
if(player === 'paper' && computer === 'scissors')  return 'loss';
if(player === 'scissors' && computer === 'paper')  return 'win';
if(player === 'scissors' && computer === 'rock')  return 'loss';
```